userspace/falco/app/actions/init_inspectors.cpp

```diff
@@ -82,6 +84,10 @@ static bool populate_filterchecks(const std::shared_ptr<sinsp>& inspector,
 		filterchecks.add_filter_check(sinsp_plugin::new_filtercheck(plugin));
 		used_plugins.insert(plugin->name());
 	}
+
```

Comment:
Looking at the check above, we should make `static_fields` part of the checks too otherwise we risk adding static fields that are overlapping with builtin ones, which would make Falco behave in a completely unpredictable way during rules matching potentially.

Reply:
The static fields are enforced the `static.` prefix in libs!

Reply:
Ok through a private chat i got it. Fixed in latest commit.